config.m4

```diff
@@ -440,11 +471,21 @@ if test "$PHP_MONGODB" != "no"; then
     fi
   fi
   if test "$PHP_LIBBSON" = "no"; then
-    m4_include(src/libbson/build/autotools/CheckAtomics.m4)
-    m4_include(src/libbson/build/autotools/FindDependencies.m4)
-    m4_include(src/libbson/build/autotools/m4/ac_compile_check_sizeof.m4)
-    m4_include(src/libbson/build/autotools/m4/ac_create_stdint_h.m4)
-    AC_CREATE_STDINT_H([$srcdir/src/libbson/src/bson/bson-stdint.h])
+    sinclude(src/libbson/build/autotools/CheckAtomics.m4)
+    sinclude(src/libbson/build/autotools/FindDependencies.m4)
+    sinclude(src/libbson/build/autotools/m4/ac_compile_check_sizeof.m4)
+    sinclude(src/libbson/build/autotools/m4/ac_create_stdint_h.m4)
+
+    sinclude(ext/mongodb/src/libbson/build/autotools/CheckAtomics.m4)
+    sinclude(ext/mongodb/src/libbson/build/autotools/FindDependencies.m4)
+    sinclude(ext/mongodb/src/libbson/build/autotools/m4/ac_compile_check_sizeof.m4)
+    sinclude(ext/mongodb/src/libbson/build/autotools/m4/ac_create_stdint_h.m4)
+    if test -d "${srcdir}/src/libbson/"; then
+      prefixdir=${srcdir}
+    else
+      prefixdir=${srcdir}/ext/mongodb
+    fi
+    AC_CREATE_STDINT_H([$prefixdir/src/libbson/src/bson/bson-stdint.h])
```

Comment:
We need to create files in the right directory.

Reply:
Since this is after `PHP_NEW_EXTENSION()` is called, can we use `$ext_srcdir` here and avoid the conditional?